extensions/azurecore/src/utils.ts

```diff
@@ -183,10 +185,12 @@ export function updateCustomCloudProviderSettings(defaultSettings: ProviderSetti
 				Logger.info(`Custom provider settings loaded for ${cloudProvider.settings.metadata.displayName}`);
 			}
 			void vscode.window.showInformationMessage(localize('providerSettings.success', 'Successfully loaded custom endpoints from settings'));
+			TelemetryReporter.sendTelemetryEvent('loadCustomEndpointsSuccess');
 
 		} catch (error) {
 			void vscode.window.showErrorMessage(localize('providerSettings.error', 'Could not load endpoints from settings, please check the logs for more details.'));
 			console.error(error.message);
+			TelemetryReporter.sendTelemetryEvent('loadCustomEndpointsError');
```

Comment:
Don't want to log the error here? Need to be careful of what we log for PII reasons - but even the callstack of the error could be useful in figuring out where the issue is.